### PR TITLE
Add moderation guidelines to subforum

### PR DESCRIPTION
**Pull request by Will-Howard**
Screenshot 2022-11-30 at 13 59 18

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203465623979622) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/comments/ModerationGuidelines/ModerationGuidelinesEditForm.tsx [hunk 1]
```diff
@@ -46,26 +49,25 @@ const ModerationGuidelinesEditForm = ({ postId, onClose, classes }: {
       </DialogTitle>
       <DialogContent>
         <Components.Typography variant="body2">
-          Edit the moderation guidelines specific to this post:
+          Edit the moderation guidelines specific to this {commentType}:
         </Components.Typography>
-        {/* TODO: fix unerlying issues so we don't need this weird addFields hack. Fields does not parse properly for non-admins */}
```
Comment (Will-Howard) [inline packages/lesswrong/components/comments/ModerationGuidelines/ModerationGuidelinesEditForm.tsx:51]: This appears to have been fixed

File: packages/lesswrong/components/comments/ModerationGuidelines/ModerationGuidelinesBox.tsx [hunk 2]
```diff
@@ -122,26 +136,25 @@ const ModerationGuidelinesBox = ({classes, post}: {
...
   const displayedGuidelines = expanded ? combinedGuidelines : truncatedGuidelines
 
   const expandable = combinedGuidelines.trim().length !== truncatedGuidelines.trim().length
 
   return (
     <div className={classes.root} onClick={expandable ? handleClick : undefined}>
-      { // FIXME: Users.canModeratePost depends on some fields that aren't always
-        // reasonable to include in fragments, so on non-post pages the edit button
-        // may be missing from moderation guidelines.
-        // @ts-ignore
```
Comment (Will-Howard) [inline packages/lesswrong/components/comments/ModerationGuidelines/ModerationGuidelinesBox.tsx:143]: This has also been fixed

File: packages/lesswrong/server/resolvers/tagResolvers.ts [hunk 3]
```diff
@@ -379,11 +379,11 @@ async function buildContributorsList(tag: DbTag, version: string|null): Promise<
       const selfVoteAdjustment = selfVoteScoreAdjustmentByUser[userId]
       const excludedPower = selfVoteAdjustment?.excludedPower || 0;
       const excludedVoteCount = selfVoteAdjustment?.excludedVoteCount || 0;
-      
+
       return {
         contributionScore: totalRevisionScore - excludedPower,
         numCommits: revisionsByThisUser.length,
-        voteCount: sumBy(revisionsByThisUser, r=>r.voteCount) - excludedVoteCount,
+        voteCount: sumBy(revisionsByThisUser, r=>r.voteCount ?? 0) - excludedVoteCount,
```
Comment (Will-Howard) [inline packages/lesswrong/server/resolvers/tagResolvers.ts:386]: I hit a bug where `voteCount` could be NaN

Comment (oetherington) [reply packages/lesswrong/server/resolvers/tagResolvers.ts:386]: lol - I feel like this indicates an issue somewhere else (unless it's just old crufty data?)

**Review by oetherington — APPROVED** [review]
LGTM!

**Pull request merged**